### PR TITLE
add mastodon image and details

### PR DESCRIPTION
**Pull request by bbrcknl**
Signed-off-by: Birgit Brecknell <birgit@sel4.systems>

### CODE REVIEW

**Comment by bbrcknl** [comment]
@june-andronick
The link to http://www2.technologyreview.com/tr35/profile.aspx?trid=1114 is timing out. Is there another link for "MIT Tech Review as a Top Innovator under 35," in your Governing Board profile?

File: content/contact/home.pml [hunk 1]
```diff
@@ -18,7 +18,7 @@
 <ul>
 <li><a href="https://lists.sel4.systems/postorius/lists/announce.sel4.systems/">Announce mailing list</a> announcements about seL4 (low volume list)</li>
 <li>LinkedIn: <a href="https://www.linkedin.com/company/sel4/">seL4 Foundation</a></li>
-<li>Twitter: <a href="https://twitter.com/seL4Foundation/">@seL4Foundation</a></li>
+<li>Mastodon: <a href="https://fosstodon.org/@sel4">[Mastadon handle]</a><br>We are suspending our use of Twitter until further notice. We will instead provide updates on Mastodon.</li>
```
Comment (gernotheiser) [inline content/contact/home.pml:21]: Please replace "[Mastadon handle]" by "@sel4@fosstodon.org"

**Comment by gernotheiser** [comment]
> @june-andronick
> The link to http://www2.technologyreview.com/tr35/profile.aspx?trid=1114 is timing out. Is there another link for "MIT Tech Review as a Top Innovator under 35," in your Governing Board profile?

This might be a temporary failure, let's just go ahead for now.

**Review by gernotheiser — APPROVED** [review]

**Comment by june-andronick** [comment]
The TR35 link doesn't seem to be responding right now. I just found the following which we could use if it's still an issue at the next PR: 
https://www.technologyreview.com/innovator/june-andronick/

**Pull request merged**